website/content/docs/v0.4/Getting Started/create-workload.md

```diff
@@ -35,7 +35,7 @@ For instance:
 ```bash
 export CONTROL_PLANE_SERVERCLASS=any
 export WORKER_SERVERCLASS=any
-export TALOS_VERSION=v0.13.0
+export TALOS_VERSION=v0.13.3
```

Comment:
this is _config_ version, not explicitly Talos version iirc, so it doesn't have to be bumped

Reply:
updated so that `make talos-artifacts` downloads the stable. This also fixed the issue with talosctl not able to destroy created machines